packages/neon/neon/lib/src/router.dart

```diff
@@ -60,13 +56,17 @@ class AccountSettingsRoute extends GoRouteData {
       name: 'Settings',
       routes: [
         TypedGoRoute<NextcloudAppSettingsRoute>(
-          path: ':appid',
+          path: 'apps/:appid',
           name: 'NextcloudAppSettings',
         ),
         TypedGoRoute<AccountSettingsRoute>(
           path: 'account/:accountid',
           name: 'AccountSettings',
         ),
+        TypedGoRoute<AddAccountRoute>(
+          path: 'addAccount',
+          name: 'addAccount',
```

Comment:
```suggestion
          path: 'account/add',
          name: 'AddAccount',
```

The account ID can never be `add` so it should be fine and also looks a lot better imo.

Reply:
I started with it but I had a problem where it would still activate the accountsettings with add as an account id and then failing to find the account.

Having the addAccount route before the accountSettings one in the routes list would probably fix this (didn't check that prossibility).

That's also the reason why I moved `settings/:appID` to `settings/apps/:appID`.
I also prefer the other way so I'll look into it.

Reply:
Ok as expected this does work.

Should I also revert the `settings/apps/:appID` or not?

Reply:
Great, but let's stick to the current way

Reply:
current being the state of this PR or the state of master?
I'm a bit confused.

Reply:
This PR, sorry for the confusion.